src/main/java/at/hannibal2/skyhanni/features/garden/pests/PestFinder.kt

```diff
@@ -154,6 +162,15 @@ object PestFinder {
 
     private var lastKeyPress = SimpleTimeMark.farPast()
 
+    @SubscribeEvent
+    fun onChat(event: LorenzChatEvent) {
+        if (!config.noPestTitle) return
+
+        val containsCheck = GardenAPI.inGarden() && noPestsChatPattern.matches(event.message)
+
+        if (event.blockedReason == "gardenPest" || containsCheck) LorenzUtils.sendTitle("§eNo pests!", 2.seconds)
```

Comment:
im like 99% sure that even if the message is blocked this event will still recieve it and match the pattern, therefore there is no need to check the blocked reason, also probably return if not on garden before checking if the config option is enabled

Reply:
indeed correct, didn't think about it, my first thought was the blockedReason path. Thank you for the changes, will be pushing again.